tests/lib/rules/i18n-no-collapsible-whitespace.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @fileoverview Disallow collapsible whitespace in translatable strings
+ * @author Automattic
+ * @copyright 2016 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require( '../../../lib/rules/i18n-no-collapsible-whitespace' ),
+	config = { env: { es6: true } },  // support for string templates
+	formatMessage = require( '../../util/format-message' ),
+	RuleTester = require( 'eslint' ).RuleTester;
+
+//------------------------------------------------------------------------------
+// Constants
+//------------------------------------------------------------------------------
+
+const NO_NEWLINES = formatMessage( rule.ERROR_MESSAGE, { problem: ' (\\n)' } );
```

Comment:
I guess now that we're running Node 4+ in the project, `const` should be okay. That said, I'd still prefer consistently at least within the same file, where there are `var` declarations above that could be `const`.